hack/update-codegen.sh

```diff
@@ -37,6 +37,7 @@ API_KNOWN_VIOLATIONS_DIR="${API_KNOWN_VIOLATIONS_DIR:-"${KUBE_ROOT}/api/api-rule
 OUT_DIR="_output"
 BOILERPLATE_FILENAME="hack/boilerplate/boilerplate.generatego.txt"
 APPLYCONFIG_PKG="k8s.io/client-go/applyconfigurations"
+PLURAL_EXCPTIONS="Endpoints:Endpoints"
```

Comment:
```suggestion
PLURAL_EXCEPTIONS="Endpoints:Endpoints"
```

I think you carried over the typo

Reply:
Fixed!